source/CMakeLists.txt

```diff
@@ -37,8 +37,6 @@ if ("${SLIRP_FOUND}" STREQUAL "")
   endif()
 endif()
 
-find_package(Boost REQUIRED)
```

Comment:
boost is now only pulled in to the `common2` library, and only if building the files dependent on it.